src/App.tsx

```diff
@@ -1,93 +1,279 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import {
+  useState, useEffect, useMemo, useCallback,
+} from 'react';
+import { UserWarning } from './component/UserWarning';
+import { client } from './utils/fetchClient';
+import {
+  getTodos, getPostTodos, deleteTodos,
+} from './api/todos';
+import { Todo } from './types/Todo';
+import { NewTodo } from './types/NewTodo';
+import { TodoList } from './component/TodoList';
+import { TodoHeader } from './component/TodoHeader';
+import { TodoFooter } from './component/TodoFooter';
+import { SortEnum } from './types/sort';
+
+const USER_ID = 10589;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [select, setSelect] = useState(SortEnum.ALL);
+  const [error, setError] = useState('');
+  const [querySearch, setQuerySearch] = useState('');
+  const [processing, setProcessing] = useState<number[]>([]);
+
+  const uncompletedLength = todos.filter(({ completed }) => !completed).length;
```

Comment:
Don't forget about camelCase 
```suggestion
  const unCompletedLength = todos.filter(({ completed }) => !completed).length;
```